cluster/src/dunit/scala/io/snappydata/cluster/SplitSnappyClusterDUnitTest.scala

```diff
@@ -501,6 +501,10 @@ object SplitSnappyClusterDUnitTest
     val countdf1 = snc.sql("select * from PR_TABLE4")
     count = countdf1.count()
     assert(count == 1000, s"Unexpected count = $count, expected 1000")
+
+    // force the stats to be populated
+    SnappyTableStatsProviderService.getService.getTableStatsFromService("APP.PR_TABLE3")
+    SnappyTableStatsProviderService.getService.getTableStatsFromService("APP.PR_TABLE4")
```

Comment:
Forcing stats population in emebedded cluster so that when connector cluster retrieves the stats all the stats are available.